app/assets/stylesheets/styles/components/guides/components/_actions_modal.css.scss

```diff
@@ -0,0 +1,33 @@
+.actions-modal {
+  -webkit-transition: opacity 200ms ease-in;
+  -moz-transition: opacity 200ms ease-in;
+  transition: opacity 200ms ease-in;
+
+  .actions ul li{
+    width: 31.5%;
+    display: inline-block;
+    vertical-align: top;
+    margin-left: 1.5%;
+  }
+}
+
+.actions-modal:target {
+  opacity:1;
+}
+
+.actions-modal > div {
+  padding: 1rem;
+  border-radius: 5px;
+  background: #f1f1f1;
```

Comment:
Color literals like `#f1f1f1` should only be used in variable declarations; they should be referred to via variable everywhere else.